File: Items.java
package AdventureText;

public class Items {
    
	private String name;
	private String description; 
	
	public Items() {
		
	}
	
	public Items(String name, String description) {
		this.name = name;
		this.description = description;
	}
	
	////getters and setters
	
	public String getName() {
		return name;
	}
	
	public void setname(String name) {
		this.name=name;
		
	}
	
	public String getDesciption() {
		return description;
	}
	
	public void setDescription(String description) {
		this.description = description;
	}
	
	
	public String toString() {
		return "Item = " + name + ", description of item = " + description;
	}
	
}

File: Items.js
class Items {
    constructor(name, description) {
      this.name = name;
      this.description = description;
    }
    
    getName() {
      return this.name;
    }
    
    setName(name) {
      this.name = name;
    }
    
    getDescription() {
      return this.description;
    }
    
    setDescription(description) {
      this.description = description;
    }
    
    toString() {
      return "Item = " + this.name + ", description of item = " + this.description;
    }
  }
  
  